Net.studio08.xbplay

alexwarddev1230 opened the pull request:
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

barthalion commented on net.studio08.xbplay.metainfo.xml:
```diff
@@ -0,0 +1,55 @@
+    <name>xbPlay: Remote Play for Xbox</name>
+    <summary>Remote play from your console or xCloud</summary>
+    <metadata_license>CC-BY-SA-4.0</metadata_license>
+    <project_license>Apache-2.0</project_license>
```
Out of curiosity, where do I find source code?

alexwarddev1230 replied:
Still in a private repo at this point. Planning to make it public in the near future.

hfiguiere replied:
So the license is a lie. Since there is no source code.

barthalion replied:
To be fair, I don't think Apache-2.0 technically requires the source to be available.

barthalion commented on net.studio08.xbplay.metainfo.xml:
```diff
@@ -0,0 +1,55 @@
+        <p>
+            Remote play from your Xbox One or Series X/S game console directly to your Linux device.<br></br>
+            It supports 1080p quality, backward compatible games, and xCloud titles. Haptic feedback is also supported on connected controllers. The xbPlay app is available on Google Play, Google Play TV, the iOS App Store, and the Amazon Marketplace (TV).<br></br>
+            *This is the linux port of the xbPlay app.
```
Not sure there's any point in mentioning it's a Linux port.

alexwarddev1230 replied:
Ok removed 👍

barthalion commented on net.studio08.xbplay.yml:
```diff
@@ -0,0 +1,63 @@
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/main/com.studio08.xbplay "$@"
+      - type: file
+        path: icon_32.png
+      - type: file
+        path: icon_64.png
+      - type: file
+        path: icon_128.png
+      - type: file
+        path: icon_256.png
```
```suggestion
```

alexwarddev1230 replied:
Help me understand why I should remove these icons? Are they just not necessary?

barthalion replied:
Yeah, 512x512 will be downscaled to 128x128 automatically.

alexwarddev1230 replied:
Understood, thanks for the details. I have removed all icons except for the 512 icon.

barthalion commented on net.studio08.xbplay.yml:
```diff
@@ -0,0 +1,63 @@
+      - chmod +x /app/main/com.studio08.xbplay
+
+      # add icons
+      - install -Dm644 icon_32.png /app/share/icons/hicolor/32x32/apps/net.studio08.xbplay.png
+      - install -Dm644 icon_64.png /app/share/icons/hicolor/64x64/apps/net.studio08.xbplay.png
+      - install -Dm644 icon_128.png /app/share/icons/hicolor/128x128/apps/net.studio08.xbplay.png
+      - install -Dm644 icon_256.png /app/share/icons/hicolor/256x256/apps/net.studio08.xbplay.png
```
```suggestion
```

alexwarddev1230 replied:
Is there a reason I shouldn't include these icons?

barthalion commented on net.studio08.xbplay.yml:
```diff
@@ -0,0 +1,63 @@
+      - install -Dm755 -t /app/bin/ run.sh
+    sources:
+      - type: archive
+        url: https://www.dropbox.com/s/333ffftl6qimhdq/data_1.1.0.zip?dl=1
```
Any reason you can't include icons, screenshots, desktop, and appdata files into that zip?

alexwarddev1230 replied:
Easier for me to track keeping assets in this repo and the zip just for the binary. Is there a reason I can't have screenshots for flathub in the flathub repo that I have?

barthalion commented:
bot, build net.studio08.xbplay

hfiguiere commented on net.studio08.xbplay.metainfo.xml:
```diff
@@ -0,0 +1,54 @@
+    <metadata_license>CC-BY-SA-4.0</metadata_license>
+    <project_license>Apache-2.0</project_license>
+    <developer_name>Studio08 Development</developer_name>
+    <url type="homepage">https://www.reddit.com/r/xbPlay/</url>
```
why isn't this a link to the homepage?

alexwarddev1230 replied:
Updated to homepage.

alexwarddev1230 commented:
@barthalion / @hfiguiere  I believe I have addressed all of the comments. Please let me know if there is anything else required before releasing. Thanks.

barthalion commented:
bot, build net.studio08.xbplay

alexwarddev1230 commented:
> Build [30871 successful](https://buildbot.flathub.org/#/builders/6/builds/30871) To test this build, install it from the testing repository:
> 
> ```
> flatpak install --user https://dl.flathub.org/build-repo/13516/net.studio08.xbplay.flatpakref
> ```

I ran the above command on my linux machine and it looks to be working fine. Anything else you need before approving for merge? Thanks

alexwarddev1230 commented:
bot, build net.studio08.xbplay

barthalion commented:
/merge
